docs/reference/transform/limitations.asciidoc

```diff
@@ -6,9 +6,12 @@
 <titleabbrev>Limitations</titleabbrev>
 ++++
 
-The following limitations and known problems apply to the {version} release of 
-the Elastic {transform} feature. The limitations are grouped into the following 
-categories:
+The following limitations and known problems apply to the {version} release of
+the Elastic {transform} feature (Note that as of {es} version 8.10.0 a new
+version number has been introduced to track changes to configuration and state
+in the transform plugin. This new version number is decoupled from the
+product version and will incrementindependently as necessary).
+The limitations are grouped into the following categories:
```

Comment:
I don't think we should say decoupling the versions is a limitation. We've done it in a transparent way such that most users don't need to care. So please revert this file to the way it was before.